### DISCUSSION TITLE
Правила отображения суммы доставки

### DISCUSSION THREAD

**Discussion by d-nikolaev** [cdek-it/wordpress#16] [Q&A] [answered]
Какие бы настройки и параметры не делали в этом блоке, стоимость доставки в корзине не меняется.
![Скриншот 25-06-2024 232731](https://github.com/cdek-it/wordpress/assets/13610260/ac609952-9799-4c2e-80eb-5cd3eacb8718)

**Comment by vermorag** [comment, marked as answer]
Добрый день. Установите последнюю вресию и проверьте с ней.
Также советую проверить, что у вас не заданы настройки для зоны, поскольку эти е правила могут быть заданы для конкретной зоны доставки (и имеют больший приоритет, чем общие правила).

Если установка последней версии не помогла - вы можете связаться с нашей техподдержкой для диагностики.

**Comment by d-nikolaev** [comment]
Установка новой версии не решила вопрос.
Данный блок почему то никак не виляет на сумму доставки.
Не важно какие опции проставить. Расчёт доставки не меняется, вернее надбавка не происходит.

**Reply by vermorag** [reply]
Добрый день. Проверьте - стоит ли настройка внутри зоны вордпресса.
